docs/building.md

```diff
@@ -0,0 +1,101 @@
+<!--
+SPDX-FileCopyrightText: 2021 TQ Tezos
+SPDX-License-Identifier: LicenseRef-MIT-TQ
+-->
+
+# Building BaseDAO
+
+BaseDAO is implemented using the [LIGO programming language](https://ligolang.org/),
+in particular its CameLIGO dialect.
+
+To generate the contract code, or the storage for one of the provided examples,
+you will need the [ligo executable](https://ligolang.org/docs/intro/installation) installed.
+
+The latest working version tested is [0.13.0](https://gitlab.com/ligolang/ligo/-/releases/0.13.0).
```

Comment:
AFAIU this this version will cause a test failure when running the tests locally. I understand that is this is probably a Morley issue, AFAIU but if someone clones the repo and run the tests, they ll get a test failure. Should we do something about it for this release?

Reply:
Good point, I forgot about it.

The latest `morley` should indeed solve this, but as I wrote in the description `morley-ledgers` needs to be updated for that to be used here.

It looks like the update should not take much, I'll try to do that and try to update the Ligo used in CI to match this version as well.

Reply:
@sras I updated `morley` to the latest release, but that doesn't contain the necessary changes yet (they are on master, but `morley-ledger` once again needs an update).

So I resorted to downgrade the ligo version used to `0.10.0` for now and updated CI accordingly.

Reply:
It works now. :+1: